routing/routing.go

```diff
@@ -203,7 +203,6 @@ type LBContext struct {
 func NewLBContext(r *http.Request, rt *Route) *LBContext {
 	return &LBContext{
 		Request: r,
-		Route:   rt,
```

Comment:
Why do we remove this?

Reply:
At the moment I don't see any reason for loadbalancer to see the information from `routing.Route`. The only reason for previous version of code were fadeIn parameters, but loadbalancer have nothing to do with them after this PR.

The only thing loadbalancer really need is slice of endpoints, which is provided in another field of LBContext. All the other stuff is not needed, so I decided to remove it for the sake of incapsulation.

Reply:
I think we can keep it and mark Route `// Deprecated`

Reply:
Also I think we need to init `Endpoints: rt.Endpoints` here until we cleanup context.

Reply:
Actually, I decided to leave `routing.Route` here because it's required for consistent hash to know both about slice of "considered" (non-fading, healthy, etc.) endpoint and slice of all endpoints.